lib/google/gax/errors.rb

```diff
@@ -143,7 +143,7 @@ def self.grpc_error_class_for(grpc_error_code)
        PermissionDeniedError, ResourceExhaustedError, FailedPreconditionError,
        AbortedError, OutOfRangeError, UnimplementedError, InternalError,
        UnavailableError, DataLossError,
-       UnauthenticatedError][grpc_error_code] || GaxError
+       UnauthenticatedError][grpc_error_code.to_i] || GaxError
```

Comment:
why did this change?

Reply:
I had some test error object that had a code method, but it returned a non-integer value. So this is a bit of protection.